cli/internal/fs/turbo_json.go

```diff
@@ -22,38 +22,40 @@ type TurboJSON struct {
 	RemoteCacheOptions RemoteCacheOptions `json:"remoteCache,omitempty"`
 }
 
-// ReadTurboConfig toggles between reading from package.json or turbo.json to support early adopters.
+const CONFIG_FILE = "turbo.json"
+
+// ReadTurboConfig toggles between reading from package.json or the CONFIG_FILE to support early adopters.
 func ReadTurboConfig(rootPath AbsolutePath, rootPackageJSON *PackageJSON) (*TurboJSON, error) {
```

Comment:
I would prefer to remove `rootPackageJSON ` and read package.json internally, but the parent function is also reading it, so we'd duplicate the read. If we only needed to read `package.json` when `turbo.json` was missing, I think it would be a good tradeoff, but we are _always_ reading `package.json` so we can log when `turbo` key exists and should be removed.

I would like to remove it, so we don't have to reproduce it a package.json for every test case we write and pass it in.

Reply:
We always need to read every `package.json` anyways.

Reply:
yeah, I was hoping to inline, but only if the double read would only apply in the fallback scenario (i.e. not very often). But since it's always used in this function, I can't make that tradeoff